gdal/swig/python/gdal-utils/osgeo_utils/samples/gdallocationinfo.py

```diff
@@ -0,0 +1,383 @@
+                             "otherwise a warning will be issued (unless quiet).")
+
+    parser.add_argument("-interp", dest="resample_alg", action="store_true",
+                        help="If set, a Bilinear interpolation would be used, otherwise the NearestNeighbour sampling.")
+
+    parser.add_argument("-b", dest="band_nums", metavar="band", type=int, nargs='+',
+                        help="Selects a band to query. Multiple bands can be listed. By default all bands are queried.")
+
+    parser.add_argument('-overview', dest="ovr_idx", metavar="overview_level", type=int,
+                        help="Query the (overview_level)th overview (overview_level=1 is the 1st overview), "
+                             "instead of the base band. Note that the x,y location (if the coordinate system is "
+                             "pixel/line) must still be given with respect to the base band.")
+
+    parser.add_argument("-l_srs", dest="srs", metavar='srs_def', type=str,
+                        help="The coordinate system of the input x, y location.")
+
+    parser.add_argument("-geoloc", dest="geoloc", action="store_true",
+                        help="Indicates input x,y points are in the georeferencing system of the image.")
+    parser.add_argument("-llgeoloc", dest="llgeoloc", action="store_true",
+                        help="Indicates input x,y points are in the long, lat (geographic) "
+                             "based on the georeferencing system of the image.")
+    parser.add_argument("-wgs84", dest="wgs84", action="store_true",
+                        help="Indicates input x,y points are WGS84 long, lat.")
+    parser.add_argument("-epsg", dest="epsg", metavar='epsg', type=int,
+                        help="Indicates input x,y points are in the given EPSG georeferencing system.")
+
+    parser.add_argument("-oo", dest="open_options", metavar="NAME=VALUE",
+                        help="Dataset open option (format specific).",
+                        nargs='+')
+
+    parser.add_argument("filename_or_ds", metavar="filename", type=str,
+                        help="The source GDAL raster datasource name.")
+
+    parser.add_argument("xy", metavar="x y", nargs='*', type=float,
+                        help="series of X Y pairs of location of target pixel. "
+                             "By default the coordinate system "
+                             "is pixel/line unless -l_srs, -wgs84 or -geoloc supplied.")
+
+    args = parser.parse_args(argv[1:])
+
+    if len(argv) == 1:
+        parser.print_help()
+        return 1
+    else:
+        interactive_mode = len(args.xy) <= 1
+        if not interactive_mode:
+            args.x = np.array(args.xy[0::2])
+            args.y = np.array(args.xy[1::2])
+
+        if args.geoloc:
+            args.srs = LocationInfoSRS.SameAsDS_SRS
+        elif args.llgeoloc:
+            args.srs = LocationInfoSRS.SameAsDS_SRS_GeogCS
+        elif args.wgs84:
+            args.srs = 4226
```

Comment:
```suggestion
            args.srs = 4326
```
This is the reason of the failure on Ubuntu 18.04 32bit, which uses PROJ master, and using 4226 exhibits a bug in PROJ master (see the "ERROR 1: PROJ: internal_proj_create_operations: SQLite error on " ... error message in https://github.com/OSGeo/gdal/pull/3685/checks?check_run_id=2373251189 )
You can likely drop the extra ,0 in TransformPoint()

Reply:
Oh, thanks, I wonder how it worked with the other builds, 4226 looks something completely different...